static/scss/_variables.scss

```diff
@@ -16,3 +16,10 @@ $search-filter: #D2E4F9;
 // other
 $block-padding: 20px;
 $page-padding: 40px;
+$progress-widget-bg-color: #ddd;
+$progress-widget-fg-color: #2ecc40;
```

Comment:
I think we should check if this is supposed to be the same color as `mm-brand-bg` or `mm-brand-fg` above (I suspect it is).

@roberthouse54 ?